libs/telemetry/build.gradle

```diff
@@ -10,6 +10,9 @@
  */
 
 dependencies {
+  // logging
+  api "org.apache.logging.log4j:log4j-api:${versions.log4j}"
```

Comment:
We should check once if we can add logger dependency in Telemetry `lib`. Only `core` lib has this dependency which is a special lib. Though i dont see a problem but we can re-confirm.

Reply:
I was a bit hesitant too initially, I will check with others on it. Thanks.

Reply:
The build fails if we add other dependencies. Please verify.